web/docs/advanced/jobs.md

```diff
@@ -2,81 +2,137 @@
 title: Recurring Jobs
 ---
 
-import { Required } from '@site/src/components/Required.tsx'
+import { Required } from '@site/src/components/Required'
+import { ShowForTs, ShowForJs } from '@site/src/components/TsJsHelpers'
 
-If you have server tasks that you do not want to handle as part of the normal request-response cycle, Wasp allows you to make that function a `job` and it will gain some "superpowers."
+In typical web apps, users send a request to the server and receive a response containing some data. If the time to get the data is short, the app feels responsive and smooth. But what happens if the server needs to do some longer work before it can respond?
+
+For instance, sending an email or making a slow HTTP request to an external API. In such cases, you might want to execute the task in the background, without blocking the user's request.
+
+Wasp offers support for background jobs, and they are called `job`s.
 
 Jobs will:
   * persist between server restarts
   * can be retried if they fail
-  * can be delayed until the future
+  * can be delayed until a future time
   * can have a recurring schedule!
```

Comment:
I'd use the word "Jobs" in each bullet point (instead of "They"). I would also change them to "Jobs".

We might also want to change the wording a little bit:
> Jobs have the following essential features: ...